docs/slimmer_templates.md

```diff
@@ -19,9 +19,9 @@ This also includes the Account product name in the layout header and changes the
 ## `gem_layout_account_manager`
 
 This is intended as a skeleton wrapper for account pages. It is intended to be used when we need a page rendered by another frontend app to look like a page from `govuk-account-manager-prototype`.
-This layout omits the default feedback component for GOVUK as the account pages use a different one from the rest of GOV.UK. Instead it introduces an account-specific phase banner, account feedback prompt, and an account nav component. It also purposefully omits the global bar and user satisfaction survey bar. 
+This layout omits the default feedback component for GOVUK as the account pages use a different one from the rest of GOV.UK. Instead it introduces an account-specific phase banner, account feedback prompt, and an account nav component. It also purposefully omits the global bar and user satisfaction survey bar.
 
-Like the former `gem_layout_account` template, his also includes the Account product name in the layout header and changes the logo link to the account homepage link.
+Like the former `gem_layout_account` template, this also includes the Account product name in the layout header and changes the logo link to the account homepage link.
```

Comment:
I'd merge this change with the previous commit.

Reply:
Will do, thanks 👍